OpenRA.Mods.Common/Traits/World/ShroudRenderer.cs

```diff
@@ -389,5 +290,16 @@ Sprite GetSprite(Sprite[] sprites, Edges edges, int variant)
 
 			return sprites[variant * variantStride + edgesToSpriteIndexOffset[(byte)edges]];
 		}
+
+		bool disposed;
+		public void Disposing(Actor self)
```

Comment:
I think we might want to consider implementing the usual ```IDisposable``` interface in this case, since we have actual disposable fields, and have this method simply delegate to it.